SU2_PY/SU2/eval/gradients.py

```diff
@@ -469,14 +473,24 @@ def multipoint( func_name, config, state=None, step=1e-2 ):
     # ----------------------------------------------------
     
     # will run in ADJOINT/
-
+    
     config.AOA = aoa_list[0]
     config.SIDESLIP_ANGLE = sideslip_list[0]
     config.MACH_NUMBER = mach_list[0]
     config.REYNOLDS_NUMBER = reynolds_list[0]
     config.FREESTREAM_TEMPERATURE = freestream_temp_list[0]
     config.FREESTREAM_PRESSURE = freestream_press_list[0]
-    config.TARGET_CL = target_cl_list[0]   
+    config.TARGET_CL = target_cl_list[0]
+    orig_marker_outlet = config['MARKER_OUTLET']
```

Comment:
It would be really great if we had the ability to easily customize the types of quantities that we use for the multipoint. As written in the framework, every time you want to use a new quantity (such as the outlet values that I had added before), you need to manually add some of these options to the python scripts. If you have some ideas for that, I am all ears.

Reply:
I agree. This is not the easiest way to go about it. I think combining this with an overall effort of consolidating the IO python scripts and the in-memory ones might be the way to go. 

I haven't really played with those python wrappers but it might be time to do that. I would imagine writing new python scripts for shape optimization and some of the other functionality that utilized the new python wrapper would probably lend it more flexibility.

Specifically we might be able to better address some of the scaling issues that we see in #733 and perform some smart normalizations. I also imagine that we could handle solution divergences intelligently to give the optimizer some useful information even though the CFD solver doesn't converge the case. This could make the optimization process a little more robust.